test/test_datapipe.py

```diff
@@ -1288,29 +1290,42 @@ def test_traverse_forked(self):
 
 
 class TestSharding(TestCase):
+
+    def get_pipeline(self):
```

Comment:
nit: Add an underscore in front of this function can prevent unittest over itself.